### PR TITLE
Increase cleanup script frequency

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -1123,7 +1123,7 @@ periodics:
       secret:
         secretName: auth-e2e-rbac-kubeconfig
```
Comment (yutongz) [inline prow/config.yaml]: Actually, I should mention it in your latest PR. If we are not supporting 0.2, we can delete these clusters for old auth/pilot/mixer repo and of course, we don't need to cleanup them.

File: prow/config.yaml [hunk 2]
```diff
@@ -1155,7 +1155,7 @@ periodics:
       secret:
         secretName: pilot-e2e-rbac-kubeconfig
```
Comment (yutongz) [inline prow/config.yaml]: Same here

File: prow/config.yaml [hunk 3]
```diff
@@ -1187,7 +1187,7 @@ periodics:
       secret:
         secretName: mixer-e2e-rbac-kubeconfig
```
Comment (yutongz) [inline prow/config.yaml]: Same here.

File: prow/test-infra-cleanup-GKE.sh [hunk 4]
```diff
@@ -27,4 +27,4 @@ set -u
 set -x
 
 echo "=== Cleaning up GKE clusters ==="
-./scripts/cleanup-GKE "$@"
+./scripts/cleanup-GKE -h 1
```
Comment (yutongz) [inline prow/test-infra-cleanup-GKE.sh:30]: I thought we agree 2 hours but I guess 1 is fine :)

Comment (sebastienvas) [reply prow/test-infra-cleanup-GKE.sh:30]: the problem is that the e2e-smoke also creates cluster during presubmit, and they run more ofter so I rather have 1 h

File: prow/config.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/config.yaml]: done

File: prow/config.yaml (hunk 2 above)
Comment (sebastienvas) [reply prow/config.yaml]: done

File: prow/config.yaml (hunk 3 above)
Comment (sebastienvas) [reply prow/config.yaml]: done

**Review by yutongz — APPROVED** [review]
/lgtm

**Pull request merged**